nodewatcher/modules/frontend/ip_space/static/js/point.js

```diff
@@ -0,0 +1,86 @@
+//Author: https://github.com/ryan-williams
```

Comment:
So we want more than just author information. Where is code coming from?

Reply:
Added repository it is based from.